### PR TITLE
feat: add create and run migrations

### PR DESCRIPTION
**Pull request by brent-hoover**
Signed-off-by: Brent Hoover <brent@thebuddhalodge.com>

### CODE REVIEW

**Comment by delagroove** [comment]
@MohanNarayana is you migrations guide around? we might use some of that prose.

**Comment by brent-hoover** [comment]
I think I already incorporated @MohanNarayana documentation here

**Comment by MohanNarayana** [comment]
@delagroove Yes, my doc is incorporated already.

File: guides/create-and-run-migrations.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Add the migration in api-migrations
+- Add the new track to `api-migrations/migrator.config.js` following the existing syntax. The app looks at this file to pick the desired version and checks against the migrations table in the DB to identify the current version.
+- Install the plugin package using `npm install @reactioncommerce/<api-plugin-package>`
+
+These steps are explained in more detail [here](https://github.com/reactioncommerce/migrator#how-to-publish-a-package-with-migrations), and you can look at the [simple-authorization](https://github.com/reactioncommerce/plugin-simple-authorization) plugin code for an example to follow.
+
+IMPORTANT: If the plugin you added a migration to is one that is built in to the stock Reaction API releases, then at the same time you bump the plugin package version in https://github.com/reactioncommerce/reaction `trunk` branch, you must also update the data version in the trunk branch of `migrator.config.js` in this repo.
```
Comment (MohanNarayana) [inline guides/create-and-run-migrations.md]: Add formatter '>' for a highlighted box.

**Review by MohanNarayana — APPROVED** [review]
Looks good

**Comment by brent-hoover** [comment]
Closing to be reopened in a later phase

**Pull request closed without merging**